app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
+    end
```

Comment:
@epwinchell There are real native SVG versions here: http://design.jboss.org/hawkular/
I am also not sure how to return a svg in one case and png in another. But looks like there is also a WildFly svg icon here http://design.jboss.org/wildfly/logo/final/wildfly_icon.svg. For EAP we still need to wait -- can you create a dummy please?